zerver/views/registration.py

```diff
@@ -363,28 +377,76 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
-                 # we have to set it here.
-                 'creating_new_team': realm_creation,
-                 'password_required': password_auth_enabled(realm) and password_required,
-                 'require_ldap_password': require_ldap_password,
-                 'password_auth_enabled': password_auth_enabled(realm),
-                 'root_domain_available': is_root_domain_available(),
-                 'default_stream_groups': get_default_stream_groups(realm),
-                 'accounts': get_accounts_for_email(email),
-                 'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
-                 'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
-                 'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
-                 'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
-                 }
+        context = {
+            'form': form,
+            'email': email,
+            'key': key,
+            'full_name': request.session.get('authenticated_full_name', None),
+            'lock_name': name_validated and name_changes_disabled(realm),
+            # password_auth_enabled is normally set via our context processor,
+            # but for the registration form, there is no logged in user yet, so
+            # we have to set it here.
+            'creating_new_team': realm_creation,
+            'billing_enabled': settings.BILLING_ENABLED,
+            'password_required': password_auth_enabled(realm) and password_required,
+            'require_ldap_password': require_ldap_password,
+            'password_auth_enabled': password_auth_enabled(realm),
+            'root_domain_available': is_root_domain_available(),
+            'default_stream_groups': get_default_stream_groups(realm),
+            'accounts': get_accounts_for_email(email),
+            'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
+            'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
+            'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
+            'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
+        }
     )
 
+def notify_support_new_realm(user_profile: UserProfile, organization_type: str,
+                             repository_link: str,
+                             organization_nature: str,
+                             organization_website: str,
+                             organization_description: str) -> None:
+    if organization_type == "Open-source":
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context={
+                'realm_name': user_profile.realm.name,
+                'organization_type': organization_type,
+                'repository_link': repository_link,
+                'realm_link': user_profile.realm.uri,
```

Comment:
I don't think is what we want. We want the proper /support page link if `STAFF_SUBDOMAIN` is available as before, and if it's not, then fall back to `realm.uri`.